warehouse/integrations/deltalake/deltalake.go

```diff
@@ -31,24 +31,28 @@ const (
 	UseSTSTokens           = "useSTSTokens"
 	EnableExternalLocation = "enableExternalLocation"
 	ExternalLocation       = "externalLocation"
+	UserAgent              = "Rudderstack"
 )
 
 const (
 	provider       = warehouseutils.DELTALAKE
 	tableNameLimit = 127
 )
 
-// Reference: https://docs.oracle.com/cd/E17952_01/connector-odbc-en/connector-odbc-reference-errorcodes.html
 const (
-	tableOrViewNotFound = "42S02"
-	columnNotFound      = "42000"
-	databaseNotFound    = "42000"
-	partitionNotFound   = "42000"
```

Comment:
What happened to `tableOrViewNotFound` for example? We don't detect this error anymore?

Also, is there a more robust way to detect these errors like by error code like it was before? It would be nice to have an integration test for this and see what happens if the table is not there for example.

Reply:
That was done [last week](https://github.com/databricks/databricks-sql-go/pull/117). It might be worth chasing them to see if they manage to review it since it might break things for us. It would be a really nice addition.